sormas-api/src/main/java/de/symeda/sormas/api/EntityDto.java

```diff
@@ -128,4 +128,13 @@ public EntityDto clone() throws CloneNotSupportedException {
 
 		return (EntityDto) super.clone();
 	}
+
+	public String getCaption() {
```

Comment:
In `getCaption()` I would write `getClass() + StringUtils.SPACE + this.getUuid()` to have a useful default if the subclass does not override it.

Reply:
getCaption defaults to toString if not overwritten
but adding this to the toString makes sense